src/BenchmarkDb/DriverBase.cs

```diff
@@ -34,24 +34,36 @@ protected static void CheckResults(ICollection<Fortune> results)
             return default;
         }
 
-        public virtual Task DoWorkSync(string connectionString)
+        public virtual async Task DoWorkSync(string connectionString)

```

Comment:
Rather than this, why don't we add ```DriverBase.SupportsScenario```? That way we can avoid running for n seconds by not even spawning any tasks.

Reply:
Will change, went to the fastest solution to run the tests, now I have more time